code/user_tools/src/wrapper.cpp

```diff
@@ -5,6 +5,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <iostream>
```

Comment:
nit: place an empty line between this and the includes above since the ones above are C specific and this is C++ specific